### PR TITLE
Update hypervisor capabilities for xenserver 7

### PR DESCRIPTION
**Pull request by syed**
It looks like I missed to add the hypervisor capabilities when changing from master to 4.9. Can this be merged as well? 

@rhtyd @karuturi ?

### CODE REVIEW

**Comment by pdion891** [comment]
I think this should be push into master as well to make sure it will exist in future version.

File: setup/db/db/schema-4910to4920.sql [hunk 1]
```diff
@@ -236,3 +236,5 @@ CALL `cloud`.`IDEMPOTENT_INSERT_GUESTOS_HYPERVISOR_MAPPING`('Xenserver', '7.0.0'
 CALL `cloud`.`IDEMPOTENT_INSERT_GUESTOS_HYPERVISOR_MAPPING`('Xenserver', '7.0.0', 'Ubuntu Trusty Tahr 14.04', 256, 0);
 
 DROP PROCEDURE `cloud`.`IDEMPOTENT_INSERT_GUESTOS_HYPERVISOR_MAPPING`
+
```
Comment (rhtyd) [inline setup/db/db/schema-4910to4920.sql]: @syed 4.9.2.0 is already released, this need to go into 4.9.2.0->4.9.3.0 upgrade path.

**Comment by pdion891** [comment]
Please merge it into master as well, it also mean we need to make sure master does have the same file `setup/db/db/schema-4920to4930.sql` too.

We will do some testing around xenserver7 soon also  :-)

**Comment by rhtyd** [comment]
@syed since 4.9.2.0 is already released, please created 4.9.2.0->4.9.3.0 upgrade path and move the changes to the suggested sql file. This needs to go into master/4.10 as well /cc @karuturi

**Comment by syed** [comment]
Thanks @rhtyd for the review. I've moved it to the schema4920to4930.sql file. @karuturi Can we also push this for 4.10?

**Comment by rhtyd** [comment]
@syed simply moving this to a file is not enough. Please see Upgrade4910to4920.java and the class usage to see how upgrade paths are implemented. We'll need a Upgrade4920to4930 upgrade path, after wards you can include this sql file in that path/class.

**Comment by rhtyd** [comment]
@syed I'll work on the upgrade path today and include your fix, thanks.

**Comment by rhtyd** [comment]
This has been included in the schema file:
INSERT IGNORE INTO `cloud`.`hypervisor_capabilities`(uuid, hypervisor_type, hypervisor_version, max_guests_limit, max_data_volumes_limit, storage_motion_supported) values (UUID(), 'XenServer', '7.0.0', 500, 13, 1);

Closing this.

**Pull request closed without merging**